README.md

```diff
@@ -9,7 +9,7 @@ When you build plugins for Fastify and you want that them to be accessible in th
 1. Use the `skip-override` hidden property
 2. Use this module
 
-__Note: the v2.x series of this module covers Fastify v3. For Fastify v2
+__Note: the v2.x & v3.x series of this module covers Fastify v3. For Fastify v2
 support refert to the v1.x series.__
```

Comment:
```suggestion
 support refer to the v1.x series.__
```

Reply:
![image](https://user-images.githubusercontent.com/1404810/96794489-9a859a00-13fe-11eb-87e8-8e1e9f14f38e.png)

🙁

Reply:
One-lined it (seems most paragraphs in this document is one line anyways) and fixed the typo